actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -1728,7 +1728,7 @@ private void createUser(Request actorMessage) {
 
   private void sendSMS(Map<String, Object> userMap) {
     ProjectLogger.log("Inside Send SMS method:", LoggerEnum.INFO);
-    // removing email check SB-4009
+    // removing email check now we need to send welcome mail as well as welcome sms Ref:SB-4009
     if (!StringUtils.isBlank((String) userMap.get(JsonKey.PHONE))) {
```

Comment:
But this condition on caller side as well .  (!StringUtils.isBlank((String) userMap.get(JsonKey.PHONE)) ) 
 and change this condition to StringUtils.isNotBlank()